### PR TITLE
Add more prosthetics and their recipes

### PR DESCRIPTION
**Pull request by MNG-cataclysm**
#### Summary
Content "Add more prosthetics and recipes to make them."

#### Purpose of change
Add some more prosthetics to increase functionality and variety for the player, and some recipes to make them.

#### Describe the solution
Add basic wooden peg legs with some cushioning on the inside. These don’t work as well as a regular prosthetic, but they’re the easiest to craft and require basic materials. Antique iron prosthetics have been added too, which work quite well but are difficult to make, and are quite heavy. Wooden peg legs are autolearned at fabrication 3, while iron ones come in “Arms and Armor”.

#### Describe alternatives you've considered
Not doing this, or only having a single type of prosthetic.

#### Testing
Loaded up the game and played around with these, everything works great.

#### Additional context
I plan on adding a recipe for modern prosthetics, I just still need to find out how they are made. I’ll eventually add arm prosthetics too.

### CODE REVIEW

**Comment by dood198** [comment]
nice

**Comment by MNG-cataclysm** [comment]
> nice

Thank you.

File: data/json/items/armor/legs_prosthetics.json [hunk 1]
```diff
@@ -38,5 +38,85 @@
...
+    "material_thickness": 1,
+    "flags": [ "LEG_PROSTHETIC", "ONLY_ONE", "FRAGILE" ],
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "mutations": [ "LEFT_PEG_LEG" ] } ] },
+    "armor": [ { "encumbrance": 0, "coverage": 100, "covers": [ "leg_stub_l", "leg_l" ] } ]
+  },
+  {
+    "id": "right_leg_prosthetic_steel",
+    "type": "ARMOR",
+    "name": { "str": "steel prosthetic right leg" },
+    "description": "A prosthetic leg, used as a replacement limb for those missing one.  Dating back to the Middle Ages and the Renaissance, this prosthetic is made of iron placed over a framework, and is attached with leather straps to the leg, padding placed inside for comfort.  Made in a better fashion than more common prosthetics at the time, this limb is incredibly durable, with many noble warriors waring these into battle.  However, they were incredibly expensive, time-consuming to make, and quite heavy, leaving them out of the hands of poorer individuals.  This one is for the right leg.",
```
Comment (Brambor) [inline data/json/items/armor/legs_prosthetics.json:86]:
```suggestion
    "description": "A prosthetic leg, used as a replacement limb for those missing one.  Dating back to the Middle Ages and the Renaissance, this prosthetic is made of iron placed over a framework, and is attached with leather straps to the leg, padding placed inside for comfort.  Made in a better fashion than more common prosthetics at the time, this limb is incredibly durable, with many noble warriors wearing these into battle.  However, they were incredibly expensive, time-consuming to make, and quite heavy, leaving them out of the hands of poorer individuals.  This one is for the right leg.",
```

File: data/json/items/armor/legs_prosthetics.json [hunk 2]
```diff
@@ -38,5 +38,85 @@
...
+    "material_thickness": 2,
+    "flags": [ "LEG_PROSTHETIC", "ONLY_ONE" ],
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "mutations": [ "RIGHT_PROSTHETIC_LEG" ] } ] },
+    "armor": [ { "encumbrance": 0, "coverage": 100, "covers": [ "leg_stub_r", "leg_r" ] } ]
+  },
+  {
+    "id": "left_leg_prosthetic_steel",
+    "type": "ARMOR",
+    "name": { "str": "steel prosthetic left leg" },
+    "description": "A prosthetic leg, used as a replacement limb for those missing one.  Dating back to the Middle Ages and the Renaissance, this prosthetic is made of iron placed over a framework, and is attached with leather straps to the leg, padding placed inside for comfort.  Made in a better fashion than more common prosthetics at the time, this limb is incredibly durable, with many noble warriors waring these into battle.  However, they were incredibly expensive, time-consuming to make, and quite heavy, leaving them out of the hands of poorer individuals.  This one is for the left leg.",
```
Comment (Brambor) [inline data/json/items/armor/legs_prosthetics.json:106]:
```suggestion
    "description": "A prosthetic leg, used as a replacement limb for those missing one.  Dating back to the Middle Ages and the Renaissance, this prosthetic is made of iron placed over a framework, and is attached with leather straps to the leg, padding placed inside for comfort.  Made in a better fashion than more common prosthetics at the time, this limb is incredibly durable, with many noble warriors wearing these into battle.  However, they were incredibly expensive, time-consuming to make, and quite heavy, leaving them out of the hands of poorer individuals.  This one is for the left leg.",
```

**Review by Brambor — CHANGES_REQUESTED** [review]
typos

**Pull request merged**